### PR TITLE
Update Case Study List Images

### PR DESCRIPTION
**Pull request by ksatx**

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 52672f99cde9834266462478d537692d45203f73

https://deploy-preview-1265--stoplightio.netlify.com

**Comment by ksatx** [comment]
I have no idea why the deploy preview is showing the changelog image for Zoom.....

File: netlify/case-studies/zoom-video.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ listSubtitle: >-
   software that combines video conferencing, online meetings, chat, and mobile.
   Learn from Tim Slagle, Manager of Developer Advocacy, how they leverage
   Stoplight to save developers several hundred man-hours each year. 
-image: /images/api-testing.jpeg
+image: /images/Zoom Blue Logo.png
```
Comment (ssspear) [inline netlify/case-studies/zoom-video.md]: Zoom image missing likely due to the naming with spaces

Screen Shot 2019-11-21 at 10 21 18 AM

Comment (ksatx) [reply netlify/case-studies/zoom-video.md]: Oh I'm such a dummy, I don't know how I didn't see that before

**Comment by ksatx** [comment]
[update list images on video testimonial case studies](https://trello.com/c/jiLrEn0G/281-update-list-images-on-video-testimonial-case-studies)

**Pull request merged**